docs/prow/templates.md

```diff
@@ -0,0 +1,74 @@
+# Templates
+
+Jobs and Prow configuration are generated from templates that are in `templates` directory. There are following items there:
+
+- `config.yaml` - config file that describes files to generate
+- `templates` - directory to gather all template files
+
+### Config structure
+
+Config file is expected to have two keys:
```

Comment:
```suggestion
The config file has two keys:
```
Is expected or has?

Reply:
Currently it has, but at the same time it has to :D